integration/service/spatialindexer/ranker/ranker_factory.go

```diff
@@ -0,0 +1,22 @@
+package ranker
+
+import (
+	"github.com/Telenav/osrm-backend/integration/oasis/osrmconnector"
+	"github.com/Telenav/osrm-backend/integration/service/spatialindexer"
+)
+
+const (
+	SimpleRanker    = "SimpleRanker"
+	OSRMBasedRanker = "OSRMBasedRanker"
+)
+
+func CreateRanker(rankerType string, oc *osrmconnector.OSRMConnector) spatialindexer.Ranker {
```

Comment:
comment

Reply:
This comment is missed until today, others has already added.
Will try to fix my linter.